lib/classes/Variables.js

```diff
@@ -822,6 +827,22 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    return BbPromise.try(() => {
+      const groups = variableString.match(this.strToBoolRefSyntax);
+      if (groups.length < 2) {
+        throw new this.serverless.classes.Error(`${variableString} cannot have an empty string`);
+      }
+      const param = groups[1].trim();
+      if (param === 'false') {
+        return false;
+      } else if (!param || param === '' || param === 'null') {
```

Comment:
I wouldn't add special treatment for `'null'`, and other cases seems not possible (so I would get rid of this condition)

Reply:
Makes sense.